pkg/k8s/from_yaml.go

```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+//Package k8s
```

Comment:
Please remove this comment. Keeping a blank line at line 17 will do fine.

Reply:
Hi.
I have removed that comment. Please check now